unfetter-discover/unfetter-ui
# Issue 1327 (JS error on Assess v3 -> Baselines)

Fixes unfetter-discover/unfetter#1327

### To test:
1. Pull this branch
2. Click into Assessments Beta
3. Open JS Console in browser
4. Now click into Baselines
5. Make sure there are no JS errors in the console
